dominion/tasks.py

```diff
@@ -184,8 +199,21 @@ def build(user_id, image):
         allowed = [
             'HOST_NAME',
             'TIME_ZONE',
+            'IMAGE_ROOTFS_SIZE',
+            'ENABLEREDUCE',
+            'MENDER_ARTIFACT_NAME',
+            'MENDER_DATA_SIZE',
+            'MENDER_INVENTORY_POLL_INTERVAL',
```

Comment:
Let's check that `MENDER_INVENTORY_POLL_INTERVAL`, `MENDER_RETRY_POLL_INTERVAL` and `MENDER_UPDATE_POLL_INTERVAL` are greater than zero